std_msgs/Float64 sh_pan_pos
std_msgs/Float64 sh_lift_pos
std_msgs/Float64 elb_pos
std_msgs/Float64 wr1_pos
std_msgs/Float64 wr2_pos
std_msgs/Float64 wr3_pos